beacon-chain/core/electra/deposits.go

```diff
@@ -285,21 +276,11 @@ func ProcessPendingDeposits(ctx context.Context, st state.BeaconState, activeBal
 		return errors.Wrap(err, "could not get finalized slot")
 	}
 
-	startIndex, err := st.DepositRequestsStartIndex()
```

Comment:
deposit_requests_start_index seems deprecated in Alpaca, would be better for next work. Another state field change...